lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -1061,13 +1076,41 @@ class MailboxController extends BaseMailboxController with MailboxActionHandlerM
       case MailboxActions.openInNewTab:
         openMailboxInNewTabAction(mailbox);
         break;
+      case MailboxActions.copySubaddress:
+        try{
+          final subaddress = getSubaddress(mailboxDashBoardController.userEmail, findNodePathWithSeparator(mailbox.id, '.')!);
```

Comment:
When I copied this subaddress: `dphamhoang+subInbox hfjdhdh hfjdhdh  dhhdhdhd hfhfhf.ti disable .ddedeutuuiiiosoodsodo oodsoidoisooo dosdisodiosoid dsoidosod @linagora.com`, composer said it error email address.

<img width="881" alt="image" src="https://github.com/user-attachments/assets/9671fa1c-c875-45a6-9129-dd2dc3048c82">

Reply:
Yes space shall be encoded.

Reply:
it was correctly encoded but not recognized by the composer (`%` was a forbidden character), this is fixed now

Reply:
should have test cases for it.

Reply:
already done in `mail_address_test.dart`